app/controllers/api/v2/job_templates_controller.rb

```diff
@@ -30,10 +30,11 @@ def show
       def_param_group :job_template do
         param :job_template, Hash, :required => true, :action_aware => true do
           param :name, String, :required => true, :desc => N_('Template name')
-          param :job_name, String, :required => true, :desc => N_('Job name')
-          param :description_format, String, :required => false, :desc => N_('This template is used to generate the description. Input values ' +
-                                                                             'can be used using the syntax %{fqdn}. You may also include the ' +
-                                                                             'job name using %{job_name}.')
+          param :job_category, String, :required => true, :desc => N_('Job category')
+          param :description_format, String, :required => false, :desc => N_('This template is used to generate the description.' +
+                                                                             'Input values can be used using the syntax %{fqdn}.' +
+                                                                             'You may also include the job category and template' +
+                                                                             'name using %{job_category} and %{template_name}.')
```

Comment:
Nitpick, please add spaces after the periods, the first and second sentences would render like `'description.Input'`